src/main/java/com/faforever/moderatorclient/ui/ViewHelper.java

```diff
@@ -1093,7 +1092,7 @@ public static void buildMessagesTable(TableView<MessageFx> messageTableView, Mes
         messageTableView.setEditable(true);
         HashMap<TableColumn<MessageFx, ?>, Function<MessageFx, ?>> extractors = new HashMap<>();
 
-        TableColumn<MessageFx, Number> idColumn = new TableColumn<>("ID");
+        TableColumn<MessageFx, Integer> idColumn = new TableColumn<>("ID");
```

Comment:
Please stick with number. There are some limitations in JavaFX where you get weird errors if you use Integer. I don't recall what is was but it is hard to track.